web/router.go

```diff
@@ -315,7 +315,8 @@ func readSanitizedJSON(buf *bytes.Buffer) (string, error) {
 
 	cleaned := map[string]interface{}{}
 	for k, v := range dst {
-		if _, ok := blacklist[strings.ToLower(k)]; ok {
+		lowerk := strings.ToLower(k)
+		if _, ok := blacklist[lowerk]; ok || strings.Contains(lowerk, "password") {
```

Comment:
Can we also add this to function directly below, `redact`?

Reply:
If there's an opportunity to DRY this up, please do. There may be some challenges around coercing types.